src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -21,23 +21,26 @@
 
 (def row-height-px 56)
 
-(def disabled-text
-  (str "This workspace contains controlled access data. You can only access the contents of this workspace if you are "
-       "dbGaP authorized for TCGA data and have linked your FireCloud account to your eRA Commons account."))
+(defn- disabled-text [protected?]
+  (if protected?
+    (str "This workspace contains controlled access data. You can only access the contents of this workspace if you are "
```

Comment:
Doesn't work in this case because the string is being rendered in an anchor title which unfortunately preserves the whitespace (including embedded newline) when rendering.